ui/src/timespan/TimeSpan.tsx

```diff
@@ -16,14 +16,13 @@ import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
 import {RemoveTimeSpan, RemoveTimeSpanVariables} from '../gql/__generated__/RemoveTimeSpan';
 import {useStateAndDelegateWithDelayOnChange} from '../utils/hooks';
-import {TimeSpans} from '../gql/__generated__/TimeSpans';
 import {isSameDate} from '../utils/time';
```

Comment:
> Side note: I think it would be preferred to set newly created timespans as selected and open the popper menu for tagging and whatnot and this side case would no longer be necessary as just pressing delete and backspace / the trash icon would feel more natural. But... I can't figure out how to get that to work and without it, this feels pretty good.

I'm okay with that, it could probably work with the following steps. But I think I rather not do this because it's hacky:
- save the timespan id of the newly created timespan
- wait for full calender rerender
- search the dom for the newly created element for the timespan
- find the timespan object by id
- call setSelected() with the found data

Basically the main problem is that the onSelect event from the calender doesn't provide the newly created element and only has the full calender as event target.

Reply:
Yea, that is the issue I was running into. That is a bit of a pain. It can just stay how it is.